replay_build_scripts/upload_build_artifacts.mjs

```diff
@@ -167,10 +167,48 @@ function prepareMacOSBinaries(buildId) {
     recursive: true,
     force: true,
   });
-  fs.renameSync(
-    path.join(outdir, "Chromium.app"),
-    path.join(outdir, "Replay-Chromium.app")
+  const appPath = path.join(outdir, "Replay-Chromium.app");
+  fs.renameSync(path.join(outdir, "Chromium.app"), appPath);
+
+  const originalWorkingDir = process.env["REPLAY_ORIGINAL_WORKING_DIR"];
+  const codesignPath = path.join(
```

Comment:
I think all these `path.join` calls will throw if env variables aren't set (which they will be from `backend/scripts/chromium/build-and-upload.sh`)

more checks for the env vars before we build up the paths?  we can't rely on `path.join("","")` doing the right thing, either - that returns `"."`.

Reply:
ah thanks, I didn't think to check for other usages of this script